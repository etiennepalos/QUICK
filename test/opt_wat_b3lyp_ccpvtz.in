DFT B3LYP BASIS=cc-pVTZ cutoff=1.0e-9 denserms=1.0e-6  zmake OPTIMIZE=5 DIPOLE LOPT

  O         -0.741530        1.752130        2.896280
  H         -1.111151        0.979769        3.352290
  H         -0.920500        2.036450        1.984040

