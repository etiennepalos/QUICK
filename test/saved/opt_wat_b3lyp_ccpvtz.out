  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2022                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **             This software is subject to the terms of the Mozilla Public License, v. 2.0.            **
  **                                                                                                     **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Shajan, A.; Giese, T. J.; Cruzeiro, V.W.D.; Smith, J.; Miao, Y.; He, X.; Ayers, K.;      
  Brothers, E.; Goetz, A.W.; Merz, K.M. QUICK-22.03                                                        
  University of California San Diego, CA and Michigan State University, East Lansing, MI, 2022             
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Thu Sep 21 14:05:24 2023
| INPUT FILE :    wat.in
| OUTPUT FILE:    wat.out
| DATA FILE  :    wat.dat
| BASIS SET PATH: /mnt/home/manathun/Work/quick-342289f/install/basis

 @ Read Job And Atom

  KEYWORD=DFT B3LYP BASIS=CC-PVTZ CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE OPTIMIZE=5 DIPOLE LOPT                                                                                                                     
 ============== JOB CARD =============
 METHOD = DENSITY FUNCTIONAL THEORY
 DENSITY FUNCTIONAL = B3LYP
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 READ SAD GUESS FROM FILE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 STANDARD GRID = SG1
 GEOMETRY OPTIMIZATION ANALYTICAL GRADIENT
 REQUEST CRITERIA FOR GEOMETRY CONVERGENCE: 
      MAX ALLOWED GEO CHANGE  =  0.189E+00
      MAX GEOMETRY CHANGE     =  0.180E-02
      GEOMETRY CHANGE RMS     =  0.120E-02
      MAX GRADIENT CHANGE     =  0.100E-02
      GRADIENT NORMALIZATION  =  0.300E-03
      MAX OPTIMIZATION CYCLE  =     5
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE =    3
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-09
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-05
      GRADIENT CUTOFF  =  0.100E-06
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = CC-PVTZ,  TYPE = CARTESIAN
| BASIS FILE = /mnt/home/manathun/Work/quick-342289f/install/basis/CC-PVTZ.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = O 
 BASIS FUNCTIONS =   35
 For Atom Kind =    2
 ELEMENT = H 
 BASIS FUNCTIONS =   15

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    3    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    2    NUMBER OF NON-HYDROGEN ATOM =    1
 NUMBER OF ELECTRONS     =   10

 -- INPUT GEOMETRY -- :
    O          -0.7415       1.7521       2.8963
    H          -1.1112       0.9798       3.3523
    H          -0.9205       2.0364       1.9840

 -- DISTANCE MATRIX -- :
              1           2           3
      1     0.00000
      2     1.83324     0.00000
      3     1.83707     3.28673     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   65
 NSHELL =   22 NPRIM  =  103
 JSHELL =   22 JBASIS =   52


 @ End Reading Molecular Information 


 @ Begin Optimization Job


 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   1 OUT OF    5  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O          -0.741530       1.752130       2.896280
  H          -1.111151       0.979769       3.352290
  H          -0.920500       2.036450       1.984040

 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       11320
|   FINAL GRID POINTS    =        7917
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        8719
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =       10972

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -68.840863692      ------         0.404     1        0.00      0.40      0.00    0.1798E+01  0.4334E+00  0.1248E+02
|  2    -70.889886360  0.204902E+01       0.502     2        0.00      0.50      0.00    0.1249E+01  0.3303E+00  0.1255E+02
|  3    -74.524662948  0.363478E+01       0.502     3        0.00      0.50      0.00    0.9610E+00  0.5846E-01  0.1572E+01
|  4    -76.248240424  0.172358E+01       0.489     4        0.00      0.49      0.00    0.3245E+00  0.1612E-01  0.3824E+00
|  5    -76.445875470  0.197635E+00       0.483     5        0.00      0.48      0.00    0.4773E-01  0.2670E-02  0.5580E-01
|  6    -76.450065473  0.419000E-02       0.465     6        0.00      0.46      0.00    0.5285E-02  0.2659E-03  0.5188E-02
|  7    -76.450117399  0.519260E-04       0.435     7        0.00      0.43      0.00    0.5570E-03  0.3860E-04  0.7406E-03
|  8    -76.450122725  0.532610E-05       0.367     8        0.00      0.37      0.00    0.8643E-04  0.7523E-05  0.1322E-03
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
|  9    -76.450122742  0.172460E-07       0.485     9        0.00      0.48      0.00    0.1522E-04  0.2950E-05  0.7120E-04
| 10    -76.450122749  0.659289E-08       0.474    10        0.00      0.47      0.00    0.3704E-05  0.3000E-06  0.9920E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER  10 CYCLES
| MAX ERROR = 0.370358E-05   RMS CHANGE = 0.299998E-06   MAX CHANGE = 0.992036E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =     5.0000061441
 BETA ELECTRON DENSITY     =     5.0000061441
 ELECTRONIC ENERGY    =    -85.473001522
 CORE_CORE REPULSION  =      9.022878773
 TOTAL ENERGY         =    -76.450122749

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
----------------------------------------------------------------------------
 VARIBLES    OLD_X            OLD_GRAD        NEW_GRAD          NEW_X
----------------------------------------------------------------------------
    1X    -0.7415300566     0.0000000000    -0.0192286351    -0.6415300566
    1Y     1.7521301338     0.0000000000    -0.0181179569     1.8521301338
    1Z     2.8962802212     0.0000000000    -0.0144203271     2.9962802212
    2X    -1.1111510849     0.0000000000     0.0066151554    -1.1734166764
    2Y     0.9797690748     0.0000000000    -0.0088983189     1.0635250954
    2Z     3.3522902560     0.0000000000     0.0310731297     3.2522902560
    3X    -0.9205000703     0.0000000000     0.0126134797    -1.0205000703
    3Y     2.0364501555     0.0000000000     0.0270162759     1.9364501555
    3Z     1.9840401515     0.0000000000    -0.0166528026     2.0840401515
----------------------------------------------------------------------------
 OPTIMZATION STATISTICS:
 MAXIMUM GRADIENT ELEMENT =     0.3107312973E-01 (REQUEST =     0.1000000000E-02 )

 @ Finish Optimization for This Step


 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   2 OUT OF    5  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O          -0.641530       1.852130       2.996280
  H          -1.173417       1.063525       3.252290
  H          -1.020500       1.936450       2.084040

 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       11320
|   FINAL GRID POINTS    =        7877
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        8697
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =       10981

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -76.409036588      ------         0.509     1        0.00      0.51      0.00    0.1699E+00  0.1128E-01  0.2347E+00
|  2    -76.457231179  0.481946E-01       0.504     2        0.00      0.50      0.00    0.2447E-01  0.3356E-02  0.7715E-01
|  3    -76.456470256  -.760923E-03       0.468     3        0.00      0.47      0.00    0.3329E-01  0.1855E-02  0.4799E-01
|  4    -76.458465892  0.199564E-02       0.458     4        0.00      0.46      0.00    0.1216E-02  0.1251E-03  0.2873E-02
|  5    -76.458471833  0.594052E-05       0.413     5        0.00      0.41      0.00    0.4183E-03  0.2229E-04  0.5110E-03
|  6    -76.458474940  0.310734E-05       0.357     6        0.00      0.36      0.00    0.3614E-04  0.7068E-05  0.1024E-03
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
|  7    -76.458474948  0.814231E-08       0.488     7        0.00      0.49      0.00    0.6329E-05  0.7412E-06  0.1676E-04
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   7 CYCLES
| MAX ERROR = 0.632883E-05   RMS CHANGE = 0.741221E-06   MAX CHANGE = 0.167637E-04
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =     4.9999948484
 BETA ELECTRON DENSITY     =     4.9999948484
 ELECTRONIC ENERGY    =    -85.387043872
 CORE_CORE REPULSION  =      8.928568924
 TOTAL ENERGY         =    -76.458474948

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
----------------------------------------------------------------------------
 VARIBLES    OLD_X            OLD_GRAD        NEW_GRAD          NEW_X
----------------------------------------------------------------------------
    1X    -0.6415300566    -0.0192286351     0.0303286156    -0.7378102535
    1Y     1.8521301338    -0.0181179569     0.0215119341     1.7835603152
    1Z     2.9962802212    -0.0144203271     0.0251745399     2.9164578556
    2X    -1.1734166764     0.0066151554    -0.0146581619    -1.1270500524
    2Y     1.0635250954    -0.0088983189    -0.0137816182     1.1061781838
    2Z     3.2522902560     0.0310731297    -0.0064397830     3.2744135022
    3X    -1.0205000703     0.0126134797    -0.0156704537    -0.9705864974
    3Y     1.9364501555     0.0270162759    -0.0077303159     1.9623668857
    3Z     2.0840401515    -0.0166528026    -0.0187347569     2.1417392709
----------------------------------------------------------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =    -0.8352199624E-02 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.9628019692E-01 (REQUEST=  0.18000E-02 )
 GEOMETRY CHANGE RMS     =     0.5896715113E-01 (REQUEST=  0.12000E-02 )
 GRADIENT NORM           =     0.1861577356E-01 (REQUEST=  0.30000E-03 )

 @ Finish Optimization for This Step


 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   3 OUT OF    5  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O          -0.737810       1.783560       2.916458
  H          -1.127050       1.106178       3.274414
  H          -0.970586       1.962367       2.141739

 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       11320
|   FINAL GRID POINTS    =        7792
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        9205
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =       11662

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -76.528886304      ------         0.513     1        0.00      0.51      0.00    0.4356E+00  0.1894E-01  0.4621E+00
|  2    -76.424881142  -.104005E+00       0.509     2        0.00      0.51      0.00    0.1440E-01  0.2632E-02  0.5184E-01
|  3    -76.425455272  0.574130E-03       0.472     3        0.00      0.47      0.00    0.1054E-01  0.9553E-03  0.2232E-01
|  4    -76.425657590  0.202318E-03       0.460     4        0.00      0.46      0.00    0.4705E-02  0.3362E-03  0.8897E-02
|  5    -76.425698271  0.406810E-04       0.440     5        0.00      0.44      0.00    0.8916E-03  0.5222E-04  0.1171E-02
|  6    -76.425694798  -.347371E-05       0.390     6        0.00      0.39      0.00    0.4672E-04  0.7247E-05  0.1487E-03
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
|  7    -76.425694802  0.476892E-08       0.493     7        0.00      0.49      0.00    0.8641E-05  0.1524E-05  0.4471E-04
|  8    -76.425694801  -.115752E-08       0.477     8        0.00      0.47      0.00    0.3951E-05  0.2908E-06  0.8924E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   8 CYCLES
| MAX ERROR = 0.395088E-05   RMS CHANGE = 0.290850E-06   MAX CHANGE = 0.892414E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =     4.9999907709
 BETA ELECTRON DENSITY     =     4.9999907709
 ELECTRONIC ENERGY    =    -86.832424224
 CORE_CORE REPULSION  =     10.406729423
 TOTAL ENERGY         =    -76.425694801

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
----------------------------------------------------------------------------
 VARIBLES    OLD_X            OLD_GRAD        NEW_GRAD          NEW_X
----------------------------------------------------------------------------
    1X    -0.7378102535     0.0303286156    -0.1324570523    -0.6602665766
    1Y     1.7835603152     0.0215119341    -0.0733422730     1.8387861667
    1Z     2.9164578556     0.0251745399    -0.1444451691     2.9807464624
    2X    -1.1270500524    -0.0146581619     0.0679209162    -1.1643935418
    2Y     1.1061781838    -0.0137816182     0.1113767276     1.0718255608
    2Z     3.2744135022    -0.0064397830    -0.0508569638     3.2565955302
    3X    -0.9705864974    -0.0156704537     0.0645361361    -1.0107866849
    3Y     1.9623668857    -0.0077303159    -0.0380344546     1.9414936572
    3Z     2.1417392709    -0.0187347569     0.1953021330     2.0952686361
----------------------------------------------------------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =     0.3278014721E-01 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.7754367691E-01 (REQUEST=  0.18000E-02 )
 GEOMETRY CHANGE RMS     =     0.4749190241E-01 (REQUEST=  0.12000E-02 )
 GRADIENT NORM           =     0.1091111968E+00 (REQUEST=  0.30000E-03 )

 @ Finish Optimization for This Step


 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   4 OUT OF    5  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O          -0.660267       1.838786       2.980746
  H          -1.164394       1.071826       3.256596
  H          -1.010787       1.941494       2.095269

 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       11320
|   FINAL GRID POINTS    =        7866
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        8738
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =       11035

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -76.397746501      ------         0.511     1        0.00      0.51      0.00    0.4205E+00  0.1592E-01  0.4310E+00
|  2    -76.454749450  0.570029E-01       0.505     2        0.00      0.50      0.00    0.5535E-01  0.7221E-02  0.1638E+00
|  3    -76.450729079  -.402037E-02       0.478     3        0.00      0.48      0.00    0.7467E-01  0.3923E-02  0.1007E+00
|  4    -76.460465347  0.973627E-02       0.470     4        0.00      0.47      0.00    0.3396E-02  0.1991E-03  0.3219E-02
|  5    -76.460494066  0.287190E-04       0.436     5        0.00      0.43      0.00    0.7424E-03  0.4940E-04  0.1242E-02
|  6    -76.460499272  0.520640E-05       0.388     6        0.00      0.39      0.00    0.5137E-04  0.6846E-05  0.1689E-03
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
|  7    -76.460499280  0.792053E-08       0.492     7        0.00      0.49      0.00    0.1096E-04  0.1308E-05  0.3664E-04
|  8    -76.460499282  0.177488E-08       0.472     8        0.00      0.47      0.00    0.4878E-05  0.3256E-06  0.9398E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   8 CYCLES
| MAX ERROR = 0.487800E-05   RMS CHANGE = 0.325565E-06   MAX CHANGE = 0.939751E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =     5.0000084984
 BETA ELECTRON DENSITY     =     5.0000084984
 ELECTRONIC ENERGY    =    -85.660224877
 CORE_CORE REPULSION  =      9.199725594
 TOTAL ENERGY         =    -76.460499282

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
----------------------------------------------------------------------------
 VARIBLES    OLD_X            OLD_GRAD        NEW_GRAD          NEW_X
----------------------------------------------------------------------------
    1X    -0.6602665766    -0.1324570523     0.0040816248    -0.6835566267
    1Y     1.8387861667    -0.0733422730     0.0034668066     1.8146619877
    1Z     2.9807464624    -0.1444451691     0.0025599320     2.9644526863
    2X    -1.1643935418     0.0679209162    -0.0010718467    -1.1580585832
    2Y     1.0718255608     0.1113767276     0.0043322114     1.0498313824
    2Z     3.2565955302    -0.0508569638    -0.0095590529     3.3108777645
    3X    -1.0107866849     0.0645361361    -0.0030097781    -0.9938315934
    3Y     1.9414936572    -0.0380344546    -0.0077990180     1.9876120146
    3Z     2.0952686361     0.1953021330     0.0069991209     2.0572801779
----------------------------------------------------------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =    -0.3480448092E-01 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.5428223437E-01 (REQUEST=  0.18000E-02 )
 GEOMETRY CHANGE RMS     =     0.3112310168E-01 (REQUEST=  0.12000E-02 )
 GRADIENT NORM           =     0.5430318227E-02 (REQUEST=  0.30000E-03 )

 @ Finish Optimization for This Step


 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   5 OUT OF    5  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O          -0.683557       1.814662       2.964453
  H          -1.158059       1.049831       3.310878
  H          -0.993832       1.987612       2.057280

 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       11320
|   FINAL GRID POINTS    =        7893
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        8679
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =       10988

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -76.462236153      ------         0.509     1        0.00      0.51      0.00    0.2989E-01  0.3618E-02  0.7682E-01
|  2    -76.460557676  -.167848E-02       0.504     2        0.00      0.50      0.00    0.5124E-02  0.6820E-03  0.1482E-01
|  3    -76.460577567  0.198916E-04       0.449     3        0.00      0.45      0.00    0.5394E-02  0.3945E-03  0.1046E-01
|  4    -76.460626680  0.491128E-04       0.430     4        0.00      0.43      0.00    0.1721E-02  0.1043E-03  0.2796E-02
|  5    -76.460630451  0.377088E-05       0.398     5        0.00      0.40      0.00    0.1711E-03  0.1113E-04  0.3633E-03
|  6    -76.460633255  0.280451E-05       0.312     6        0.00      0.31      0.00    0.1146E-04  0.9254E-05  0.1924E-03
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
|  7    -76.460633259  0.359126E-08       0.488     7        0.00      0.49      0.00    0.3939E-05  0.1206E-05  0.2338E-04
|  8    -76.460633263  0.382833E-08       0.463     8        0.00      0.46      0.00    0.2064E-05  0.3122E-06  0.5652E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   8 CYCLES
| MAX ERROR = 0.206388E-05   RMS CHANGE = 0.312244E-06   MAX CHANGE = 0.565247E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =     4.9999950914
 BETA ELECTRON DENSITY     =     4.9999950914
 ELECTRONIC ENERGY    =    -85.531702868
 CORE_CORE REPULSION  =      9.071069605
 TOTAL ENERGY         =    -76.460633263

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
----------------------------------------------------------------------------
 VARIBLES    OLD_X            OLD_GRAD        NEW_GRAD          NEW_X
----------------------------------------------------------------------------
    1X    -0.6835566267     0.0040816248     0.0002487194    -0.6738620025
    1Y     1.8146619877     0.0034668066    -0.0038498711     1.8293660174
    1Z     2.9644526863     0.0025599320     0.0068023633     2.9636152718
    2X    -1.1580585832    -0.0010718467     0.0005439144    -1.1627852474
    2Y     1.0498313824     0.0043322114    -0.0037744911     1.0555295422
    2Z     3.3108777645    -0.0095590529     0.0075150952     3.2898553898
    3X    -0.9938315934    -0.0030097781    -0.0007926338    -0.9987995534
    3Y     1.9876120146    -0.0077990180     0.0076243622     1.9672098251
    3Z     2.0572801779     0.0069991209    -0.0143174585     2.0791399672
----------------------------------------------------------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =    -0.1339805348E-03 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.2185978929E-01 (REQUEST=  0.18000E-02 )
 GEOMETRY CHANGE RMS     =     0.1385009613E-01 (REQUEST=  0.12000E-02 )
 GRADIENT NORM           =     0.6632609535E-02 (REQUEST=  0.30000E-03 )

 @ Finish Optimization for This Step

 WARNING: REACHED MAX OPT CYCLES. THE GEOMETRY IS NOT OPTIMIZED.
          PRINTING THE GEOMETRY FROM LAST STEP.
============= GEOMETRY INFORMATION (NOT OPTIMIZED) ===========

 OPTIMIZED GEOMETRY IN CARTESIAN
 ELEMENT      X         Y         Z
  O       -0.6739    1.8294    2.9636
  H       -1.1628    1.0555    3.2899
  H       -0.9988    1.9672    2.0791

 FORCE
 ELEMENT      X         Y         Z
  O       -0.0001    0.0020   -0.0036
  H       -0.0003    0.0020   -0.0040
  H        0.0004   -0.0040    0.0076

 MINIMIZED ENERGY =    -76.460633263
===============================================================

 @ Finish Optimization Job


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      O        0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      H        0.97175  1     0.00000  0      0.00000  0      1    0    0
     3      H        0.95230  1   104.79745  1      0.00000  0      1    2    0

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   O             -0.3889            -0.1194
   H              0.1919             0.0590
   H              0.1970             0.0604
   TOTAL        -0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
   -1.3184   -1.0528   -0.8574    1.8925

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     0.432710000(  1.32%)
| DFT GRID OPERATIONS =     0.173621000(  0.53%)
| TOTAL SCF TIME      =    18.885615000( 57.69%)
|       TOTAL OP TIME      =    18.810307000( 57.46%)
|             TOTAL 1e TIME      =     0.023296000(  0.07%)
|             TOTAL 2e TIME      =    15.034367000( 45.93%)
|             TOTAL EXC TIME     =     3.727938000( 11.39%)
|       TOTAL DII TIME      =     0.074969000(  0.23%)
|             TOTAL DIAG TIME    =     0.031846000(  0.10%)
| TOTAL GRADIENT TIME      =    13.150222000( 40.17%)
|       TOTAL 1e GRADIENT TIME      =     0.122590000( 0.43%)
|       TOTAL 2e GRADIENT TIME      =    11.631659000(35.53%)
|       TOTAL EXC GRADIENT TIME     =     1.379198000(  4.21%)
| TOTAL TIME          =    32.736640000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes 32.7 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Thu Sep 21 14:05:57 2023
